.circleci/config.yml

```diff
@@ -84,6 +93,41 @@ jobs:
             source env/bin/activate
             make entry-point-test
 
+  package_test:
+    working_directory: ~/nlp_primitives
+    parameters:
+      image_tag:
+        type: string
+        default: "python:3.7"
+    executor:
+      name: python
+      image_tag: << parameters.image_tag >>
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/nlp_primitives
+      - package_nlp_primitives
+      - run : 
+          name: "Install From Package in Virtual Environment"
+          command: |
+            virtualenv venv
+            source venv/bin/activate
+            python -m pip config --site set global.progress_bar off
+            python -m pip install --upgrade pip
+            python -m pip install pytest
```

Comment:
Can delete this! Now that we've fixed the "test code in the package" issue in this PR.